src/plugins/discover/public/application/main/discover_main_route.tsx

```diff
@@ -74,25 +71,20 @@ export function DiscoverMainRoute(props: Props) {
   const loadDefaultOrCurrentDataView = useCallback(
     async (searchSource: ISearchSource) => {
       try {
-        const hasUserDataViewValue = await data.dataViews.hasData
-          .hasUserDataView()
-          .catch(() => false);
-        const hasESDataValue =
-          isDev || (await data.dataViews.hasData.hasESData().catch(() => false));
-        setHasUserDataView(hasUserDataViewValue);
-        setHasESData(hasESDataValue);
-
-        if (!hasUserDataViewValue) {
-          setShowNoDataPage(true);
-          return;
-        }
+        /*
+        Note: @kertal @maja
```

Comment:
@majagrubic @kertal Need some insight on this... because this check could throw the empty state into a no-op if there's no default data view, but there's data and data views... because the no data page would render null.

That's what we're seeing in Cloud, which is what led me to this change.

Reply:
This can be removed! Thx for fixing this!